internal/stack/compose.go

```diff
@@ -146,6 +146,27 @@ func dockerComposeDown(options Options) error {
 	return nil
 }
 
+func dockerComposeStatus(options Options) ([]compose.ServiceStatus, error) {
+	p, err := compose.NewProject(DockerComposeProjectName, options.Profile.FetchPath(profile.SnapshotFile))
+	if err != nil {
+		return nil, errors.Wrap(err, "could not create docker compose project")
+	}
+
+	statusOptions := compose.CommandOptions{
+		Env: newEnvBuilder().
+			withEnv(stackVariantAsEnv(options.StackVersion)).
+			withEnvs(options.Profile.ComposeEnvVars()).
+			build(),
+		Services: options.Services,
+	}
+
+	statusServices, err := p.Status(statusOptions)
+	if err != nil {
+		return nil, errors.Wrap(err, "running command failed")
```

Comment:
While running locally, I see the following error, even when all the containers are up and healthy :thinking: 
```
Error: failed updating the stack images: stack status failed: running command failed: container not recognised: /elastic-package-stack-elastic-agent-1
```

Reply:
What command did you use to boot the stack ? 
I think maybe the docker output is using different name patterns. For instance, when I run locally the stack and check the containers running these are their names:
```bash
 $ docker ps --format '{{.ID}} {{ .Names}}'
f2548695fda4 elastic-package-stack_elastic-agent_1
cd053207a69a elastic-package-stack_fleet-server_1
e5ae9f4a9ec8 elastic-package-stack_kibana_1
9f81003784b6 elastic-package-stack_elasticsearch_1
23a56affd189 elastic-package-stack_package-registry_1
```

@jsoriano What is your output of this command when you have the stack up and running ?

Reply:
> What command did you use to boot the stack ?

`elastic-package stack up -v -d`, with `docker-compose` 2.3.3.

> What is your output of this command when you have the stack up and running ?

```
$ docker ps --format '{{.ID}} {{ .Names}}'
7bcbf178400d elastic-package-stack-elastic-agent-1
3e4549a3fae3 elastic-package-stack-fleet-server-1
f84559da2eeb elastic-package-stack-kibana-1
64a987ed7d99 elastic-package-stack-package-registry-1
b1af47e723d5 elastic-package-stack-elasticsearch-1
```
There are `-` where you have `_`.

Maybe using the docker compose labels is more predictable:
```
$ docker ps --format '{{.ID}} {{.Label "com.docker.compose.service"}}'
7bcbf178400d elastic-agent
3e4549a3fae3 fleet-server
f84559da2eeb kibana
64a987ed7d99 package-registry
b1af47e723d5 elasticsearch
```

Reply:
For completeness, in my case the docker-compose version is `1.29.2`.
Keeping the regex could be eventually problematic if there are those differences between versions. I'm going to change the implementation and get the service name from the labels, as you suggested. Thanks!